cel/canonical_eventlog.go

```diff
@@ -389,5 +394,20 @@ func (c *CEL) replay(bank *pb.PCRs) error {
 		return nil
 	}
 
-	return fmt.Errorf("CEL replay failed for these PCRs in bank %v: %v", tpm2Alg, failedReplayPcrs)
+	return fmt.Errorf("CEL replay failed for these PCRs in bank %v: %v", cryptoHash, failedReplayPcrs)
+}
+
+// VerifyDigest checks the digest generated by the given record's content to make sure they are equal to
+// the digests in the digestMap.
+func VerifyDigest(c Content, digestMap map[crypto.Hash][]byte) error {
```

Comment:
Nit: This should be called `VerifyDigests` (as it verifies multiple digests)

Reply:
Done